app/docker/views/services/create/createServiceController.js

```diff
@@ -426,6 +426,12 @@ function ($q, $scope, $state, $timeout, Service, ServiceHelper, ConfigService, C
     .then(function success(data) {
       var serviceIdentifier = data.ID;
       var userId = Authentication.getUserDetails().ID;
+      console.log($scope.applicationState.endpoint)
+      var webhook = Webhook.create({ServiceID: serviceIdentifier}).$promise
```

Comment:
You'll want to create a `WebhookService` (`app/portainer/services/api/webhookService.js`) to centralize the usage of the `Webhook` rest factory. See the `TagService` for a similar example: https://github.com/portainer/portainer/blob/develop/app/portainer/services/api/tagService.js

Your issue related to the endpoint identifier being passed as a query parameter rather than in the request body is due to the fact tat `POST` requests created with `ngResource` will use an extra parameter for the body, e.g:

```javascript
Webhook.create({}, {ServiceID: ..., EndpointID: ...})
```

Again, refer to the `createTag` function for an example: https://github.com/portainer/portainer/blob/develop/app/portainer/services/api/tagService.js#L36